dump/dump.sh

```diff
@@ -1,3 +1,5 @@
 #!/usr/bin/env bash
 set -eux
-DYLD_INSERT_LIBRARIES=dump.dylib "$@"
+FILENAME=dump.dylib
+FILEPATH="$(cd "$(dirname "$FILENAME")"; pwd)/$(basename "$FILENAME")"
```

Comment:
Why did you add these? Given that FILENAME is a constant, this will always be the same value anyway.

Reply:
LD_PRELOAD requires the full path, not the relative path.

Reply:
Just use $PWD/$FILENAME then.

`dirname $FILENAME` will always be `.`, and `basename $FILENAME` will always be `$FILENAME`